.env

```diff
@@ -1,2 +1,11 @@
 OPENAI_API_KEY=1111
-OPENAI_URL='https://api.openai.com/v1/completions'
\ No newline at end of file
+OPENAI_URL='https://api.openai.com/v1/completions'
+CALENDAR_URL='https://www.googleapis.com/calendar/v3/calendars/primary/events'
+GOOGLE_EXTENSION_KEY='MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnDQJ6G55P2Zl3ab5eelg
+//4acXe7pqRf4wglQpnmmRunXFKtSv5kjSmdSGrVslNO1nas/2FlncSKYk5LieSD
+PtuWV90sTyX6V44nTM7H4o9ic11pZdCRBMRRgwOYRo6cTaYdCh7EgiJicvb2Akou
+Md7tL+KaxCY78Y4KJhhDd12XczluzB16f0lHLQ0jCSA3W9u9zPg8vMzyNK7H3cSp
+lnEsLjt0FW0DQmWB3qWBhW8nkTJ7zESmP6WU+aI07BzrwTFSpjQZVUjjjbr8wQyS
+qFIyoyUNRsOj7W+BhYNyUl5CioDQCA2kAob4VVaVwK8cfBIYwrtU9YTBNHhIKRac
+iQIDAQAB'
+GOOGLE_CLIENT_ID='180382388763-56fn6obj1btc2rpj05eim35jnleioj4d.apps.googleusercontent.com'
```

Comment:
nit: consider adding a new line at the end of each file. It is the practice because it allows easy development next time on this file. Not a big deal though. This applies for the other files too. The red mark in the Github diff means that there is no new line.